frontend/src/pages/projects/screens/projects/ProjectListView.tsx

```diff
@@ -47,7 +47,7 @@ const ProjectListView: React.FC<ProjectListViewProps> = ({ allowCreate, scope })
     setSearch('');
   };
 
-  const searchTypes = React.useMemo(() => Object.values(SearchType), []);
+  const searchTypes = React.useMemo(() => [SearchType.NAME, SearchType.USER], []);
```

Comment:
Why? Remove the unused value from the SearchType enum?

Reply:
Hmm, because I found that the `SearchType` is also used in `InferenceServiceListView`, where they need `SearchType.PROJECT`...

Reply:
@andrewballantyne OK, so I decided to change `InferenceServiceListView` to use the weird generic `DashboardSearchField` for now to make this project search field pure. We can change the use of `DashboardSearchField` in the future.

Reply:
I don't think you should have done that... that bloats this. We are talking enums here -- why are we not just making an enum for each page instead of some bastardized combination that needs to now know of what another page is doing in order to operate. Maybe we should talk about the foot print of this work.

Reply:
This is fine, talked with Juntao. Will test and see if it works as intended.